indico/modules/events/abstracts/blueprint.py

```diff
@@ -42,6 +44,8 @@
 _bp.add_url_rule('/abstracts/<int:abstract_id>/abstract.pdf', 'display_abstract_pdf_export', RHDisplayAbstractExportPDF)
 _bp.add_url_rule('/abstracts/<int:abstract_id>/attachments/<file_id>/<filename>', 'download_attachment',
                  RHAbstractsDownloadAttachment)
+_bp.add_url_rule('/abstracts/mine', 'my_abstracts', RHMyAbstracts)
+_bp.add_url_rule('/abstracts/my-abstracts.pdf', 'my_abstracts_pdf_export', RHMyAbstractsExportPDF)
```

Comment:
Wut:question: How does the file `mine.pdf` in your computer indicate that it contains abstracts? I'll stick to `my-abstracts.pdf`...